pkg/controllers/user/systemimage/systemimage.go

```diff
@@ -53,7 +59,10 @@ func (s *Syncer) Sync(key string, obj *v3.Project) (runtime.Object, error) {
 	}
 
 	changed := false
+	systemServices := getSystemService()
 	for k, v := range systemServices {
+		v.Init(s.userContext)
```

Comment:
Question, why do we init every time? maybe we check if it's been already initialized, since this is on every project sync?

Reply:
updated